Intune install method to use Settings Catalog (#11182)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/dlp-chrome-get-started.md

#### Microsoft Intune Force Install Steps

> Settings Catalog can be used now to manage the Chrome extensions.

Reviewer: ```suggestion
Using the settings catalog, follow these steps to manage Chrome extensions:
```

4. Select **Windows 10 and later** as the platform.

5. Select **Settings catalog** as the profile type.

6. Select **Custom** as the template name.

7. Select **Create**.

8. Enter a name and optional description on the **Basics** tab and select **Next**.

9. Select **Add settings** on the **Configuration settings** tab.

> 10. Select "Administrative Templates \ Google \ Google Chrome \ Extensions"
> 11. Select "Configure the list of force-installed apps and extensions"
> 12. Change the toggle to enabled
> 13. Enter the following value for the Extension/App IDs and update URL:
>      echcggldkblhodogklpincgchnpgcdco;https://clients2.google.com/service/update2/crx

Reviewer: ```suggestion
10. Select **Administrative Templates** > **Google** > **Google Chrome** > **Extensions**.
11. Select **Configure the list of force-installed apps and extensions**.
12. Change the toggle to **Enabled**.
13. Enter the following value for the extensions and app IDs and update URL:
     `echcggldkblhodogklpincgchnpgcdco;https://clients2.google.com/service/update2/crx`.
```
